# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../scons/environment.patch
# Copyright (C) 2009 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- scons-1.2.0/engine/SCons/Action.py.orig	2008-12-21 08:59:59.000000000 +0200
+++ scons-1.2.0/engine/SCons/Action.py	2009-02-28 00:06:33.000000000 +0200
@@ -561,7 +561,7 @@
 def get_default_ENV(env):
     global default_ENV
     try:
-        return env['ENV']
+        return os.environ
     except KeyError:
         if not default_ENV:
             import SCons.Environment
